src/main/java/org/opentripplanner/routing/edgetype/VehicleRentalEdge.java

```diff
@@ -38,27 +38,31 @@ public State traverse(State s0) {
                 case BEFORE_RENTING:
                     return null;
                 case HAVE_RENTED:
-                    if (options.useVehicleRentalAvailabilityInformation && stationVertex.getSpacesAvailable() == 0) {
+                    if (options.useVehicleRentalAvailabilityInformation && (
+                            stationVertex.getStation().getSpacesAvailable() == 0 || !stationVertex.getStation().isAllowDropoff()
```

Comment:
This could be simplified by adding a business utility method like 

```
if (options.useVehicleRentalAvailabilityInformation && stationVertex.getStation().allowDropOfNow()) { ....
```